### PR TITLE
CM-1095 parallel publishing mediatr notifications

### PR DESCRIPTION
**Pull request by GergoVandor**

### CODE REVIEW

File: dotnet-client-libraries/MediatrParallelPublishing/MediatrParallelPublishing/Program.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿
+using MediatR;
+using MediatR.NotificationPublishers;
+using MediatrParallelPublishing;
+using MediatrParallelPublishing.NotificationHandlers;
```
Comment (grantwinney) [inline dotnet-client-libraries/MediatrParallelPublishing/MediatrParallelPublishing/Program.cs:5]: This line appears to be unnecessary.

File: dotnet-client-libraries/MediatrParallelPublishing/MediatrParallelPublishing/Program.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿
```
Comment (grantwinney) [inline dotnet-client-libraries/MediatrParallelPublishing/MediatrParallelPublishing/Program.cs:1]: Also, this blank line can be removed.

File: dotnet-client-libraries/MediatrParallelPublishing/MediatrParallelPublishing/Program.cs (hunk 1 above)
Comment (GergoVandor) [reply dotnet-client-libraries/MediatrParallelPublishing/MediatrParallelPublishing/Program.cs]: removed

File: dotnet-client-libraries/MediatrParallelPublishing/MediatrParallelPublishing/Program.cs (hunk 2 above)
Comment (GergoVandor) [reply dotnet-client-libraries/MediatrParallelPublishing/MediatrParallelPublishing/Program.cs]: removed

**Pull request merged**